src/components/mgt-people/mgt-people.ts

```diff
@@ -130,6 +169,13 @@ export class MgtPeople extends MgtTemplatedComponent {
 
         if (this.groupId) {
           this.people = await client.getPeopleFromGroup(this.groupId);
+        } else if (this.userIds) {
+          const users = [];
+          // tslint:disable-next-line: prefer-for-of
+          for (let i = 0; i < this.userIds.length; i++) {
+            users.push(await client.getUser(this.userIds[i]));
```

Comment:
you can make these calls in parallel, should speed things up.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
Thanks! refactored: 87cbf3b